src/Backends/Cubics/UNIFAC.h

```diff
@@ -20,7 +20,7 @@ namespace UNIFAC
     private:
         CoolProp::CachedElement _T; ///< The cached temperature
 
-        std::size_t N = 0; ///< Number of components
+        std::size_t N; ///< Number of components
```

Comment:
If you did want to initialize this variable, the right place for that is the initializer for the class.